### DISCUSSION TITLE
Portable version not saving state of ShowCurrentBranchOnly

### DISCUSSION THREAD

**Discussion by steventkim** [gitextensions/gitextensions#11503] [General]
### Environment

- Git Extensions 4.2.1.17611
- Build b0c0b284876b63c52c77195ada82a85bfadbb083
- Git 2.41.0.windows.2 (recommended: 2.42.0 or later)
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.25
- DPI 96dpi (no scaling)
- Portable: True
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 3.1.22 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 3.1.32 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 5.0.14 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

### Issue description

In my Portable setup, the state of the View > Branches (AllBranches/CurrentBranchOnly/FilteredBranches) isn't getting updated in the GitExtensions.settings file.

### Steps to reproduce

1. Install Portable GE v4.1 - the state of View > Branches is remembered.
2. Upgrade to Portable v4.2 - the state is no longer remembered.

### Did this work in previous version of GitExtensions?

Yes, I have an installation of GE 4.1.0 where this feature works.

**Comment by mstv** [comment]
Since #10851, only the defaults of the View Settings are persisted. They need to be saved explicitly.

![image](https://github.com/gitextensions/gitextensions/assets/36601201/9d947d0e-5e9f-44ee-81b7-130915d4669c)
